src/components/file-uploader/FileUploader.tsx

```diff
@@ -17,16 +17,16 @@ import {
   AddDocuments,
   ButtonVariantEnum,
   ComponentEnum,
-  Emitter,
+  UploadFileEmitter,
   InputEnum,
   SizeEnum
 } from '~/types'
 
 interface FileUploaderProps {
   buttonText: string
-  emitter: ({ files, error }: Emitter) => void
+  emitter: UploadFileEmitter
   initialState: File[]
```

Comment:
If `initialState ` has a default value, could it be optional (similar to `initialError `)? It will be useful if we don't have an initial value.

Reply:
Thanks for the comment, I didn't take it into account this tiny possible typescript issue when performing this task. Already fixed